src/buffers/ClientTimeshift.h

```diff
@@ -92,5 +92,12 @@ namespace timeshift {
     }
     virtual PVR_ERROR GetStreamTimes(kodi::addon::PVRStreamTimes& times) override;
 
+    virtual bool IsRealTimeStream() const override
+    {
+      if (time(nullptr) - m_streamStart < 10 + m_prebuffer)
+        return true;
+      return false;
```

Comment:
Could be simplified to one line: `return (time(nullptr) - m_streamStart < 10 + m_prebuffer);`

Reply:
Sure